installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned.adoc

```diff
@@ -11,7 +11,10 @@ In {product-title} {product-version}, you can install a cluster on Nutanix infra
 == Prerequisites
 
 * You have reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
-* If you use a firewall, you have configured it to xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[grant access] to the sites that {product-title} requires. This includes the use of Telemetry.
+* The installation program requires access to port 9440 on Prism Central and Prism Element. You verified that port 9440 is accessible.
+* If you use a firewall, you have:
```

Comment:
For lines 15 - 17, see my suggestions for lines 20 - 22 in `installing-nutanix-installer-provisioned.adoc`.